frontend/spec/features/products_spec.rb

```diff
@@ -96,24 +96,20 @@
 
       it 'when adding a product to the cart', js: true do
         visit spree.product_path(product)
-        click_button 'Add To Cart'
+        add_to_cart(product)
```

Comment:
`#add_to_cart` helper visits a Product's page internally, so we don't need this `visit` on the 98th line

Reply:
@KirillKayumov please fix this